firebase-sessions/src/test/kotlin/com/google/firebase/sessions/SessionCoordinatorTest.kt

```diff
@@ -43,8 +43,7 @@ class SessionCoordinatorTest {
     val firebaseInstallations = FakeFirebaseInstallations("FaKeFiD")
     val sessionCoordinator =
       SessionCoordinator(
-        firebaseInstallations,
-        context = TestOnlyExecutors.background().asCoroutineDispatcher() + coroutineContext,
+        firebaseInstallations = FakeFirebaseInstallations("FaKeFiD"),
```

Comment:
Could it be `firebaseInstallations = firebaseInstallations`? Why do we recreate a new instance here?

Reply:
Fixed!